timeless/message/message_resource.py

```diff
@@ -0,0 +1,8 @@
+""" Message resource for internationalization """
+
+
+class MessageResource:
+    """ Interface that defines behavior """
+
+    def get(self, key, locale):
+        raise Exception("Message Resource does not implement methods!")
```

Comment:
@paulodamaso the way I understand it, ``MessageResource`` is the interface/abstract class, so shouldn't this method just be abstract?

Reply:
@amihaiemil Yes, it could, but I think that it brings a lot of complexity and garbage to the code, this way it becomes simpler